### DISCUSSION TITLE
How to have plain terminal output without the "🔓 password for "XXX" copied to clipboard!"

### DISCUSSION THREAD

**Discussion by esseti** [Dashlane/dashlane-cli#291] [Q&A] [answered]
I've few scripts that i use in the terminal as this when using a very old version
`fab deploy -s amazingsw -i serverone.test --sudo-psw (dcli p root-server -o password) -v 6.8.3`

now i switched to `-o console` but the message 
> 🔓 password for "Root-server" copied to clipboard!

and is sent to the command as well.

how could i still use the dcli as before? is there another way of doing it?

**Comment by esseti** [comment, marked as answer]
I reply myself `(dcli read dl://Root-server/password)` is the way. worth to notice that is case sensitive compared to the `p` command

**Reply by Mikescops** [reply]
Hello, there is a bug in the `dcli p` `-o console` command, the "copied to clipboard" should not be displayed. I'll fix it asap.

In your use case, `dcli read` is indeed the best solution indeed.

**Reply by esseti** [reply]
For the sake of consistency it should be case sensitive as well.

**Reply by Mikescops** [reply]
`dcli p` is meant to be a search while `dcli read` is a path lookup, so this behaviour is expected.

**Reply by Mikescops** [reply]
Changed my mind I made it case unsensitive for `dcli read` when used with `title` only. Btw, I highly recommend to use the `id` of the credential you want to get with dcli read. It's much more performant because using `title` forces us to decrypt the whole vault as titles are encrypted, whereas ids are not so we can get and decrypt only the right password.
